Types of Contributions

```diff
@@ -0,0 +1,264 @@
+TYPES OF CONTRIBUTIONS
+
+In this chapter, we will discuss the types of contribution that open source projects involved.
+
+Majorly, there are two types of contributions. Which are; 
+Non-coding type of contribution
+Coding type of contribution
+
+Non-Coding Type Of Contributions
+
+Non-coding contributions are valuable contributions made to open source projects that do not involve writing or modifying code. These contributions are essential for the overall health and success of open source communities and projects. They help improve documentation, user experience, project management, and community engagement.
+
+Here are some common types of non-coding contributions
+
+Documentation
+Graphic design
+Blog writing
+
+
+DOCUMENTATION
+
+Documentation is a crucial type of open source contribution that involves creating, improving, or maintaining the written resources that accompany open source software projects. Good documentation can make a project more accessible and user-friendly, and it can help attract new contributors.
+
+Contributing to documentation in open source projects is a valuable way to make a meaningful impact, even if you're not a developer. Clear and well-organized documentation enhances the overall quality of the project and fosters a more welcoming and inclusive open source community.
```

Comment:
[Proofreading] While the second paragraph is helpful in emphasizing the benefits of docs-related contributions, consider omitting it for clarity. I also recommend beginning the paragraph with a statement that draws people's interest in this specific type of OS contribution. 
```suggestion
## Documentation
Are you a lover of words? Do grammar errors irritate you like seeing pineapple on pizza? Then, documentation is for you! 
This involves creating, improving, or maintaining the written resources that accompany open source software projects. Contrary to popular belief, documentation is crucial to open source projects. It can make a project more accessible and user-friendly, and it can help attract new contributors.